katharsis-core/src/test/java/io/katharsis/queryspec/repository/DefaultQuerySpecDeserializerTest.java

```diff
@@ -133,6 +133,18 @@ public void testFilterWithDefaultOp() throws InstantiationException, IllegalAcce
 	}
 
 	@Test
+	public void testFilterWithDotNotation() throws InstantiationException, IllegalAccessException {
+		QuerySpec expectedSpec = new QuerySpec(Task.class);
+		expectedSpec.addFilter(new FilterSpec(Arrays.asList("project", "name"), FilterOperator.EQ, "value"));
+
+		Map<String, Set<String>> params = new HashMap<>();
+		add(params, "filter[project.name]", "value");
```

Comment:
done, 

the attribute names are actually already validated as long as that validation is not disabled.